src/NServiceBus.Core/Features/FeatureStartupTask.cs

```diff
@@ -8,16 +8,25 @@ public abstract class FeatureStartupTask
         /// <summary>
         /// Will be called when the endpoint starts up if the feature has been activated.
         /// </summary>
-        protected abstract void OnStart();
-        
+        [ObsoleteEx(TreatAsErrorFromVersion = "6", RemoveInVersion = "7", ReplacementTypeOrMember = "OnStart")]
+        protected virtual void OnStart()
+        {
+        }
+
+        /// <summary>
+        /// Will be called when the endpoint starts up if the feature has been activated.
+        /// </summary>
+        /// <param name="bus">Seond context.</param>
+        protected abstract void OnStart(ISendOnlyBus bus);
```

Comment:
I know we talked about this but I can't find it on the linked issue. Can we clarify on the summary when OnStart and OnStop are called (e.g. "after bus has started and before any messages from queues are processed")?